ST_stats.h

```diff
@@ -0,0 +1,50 @@
+/* Chandler Haukap- This file defines all structs used in ST_stats.c.
+ * It was created to allow the gridded code to instanciate accumulators
+ * without including ST_stats.c. 
+ * 
+ * To read the documentation related to creating this struct see issues
+ * #259 and #266 on GitHub */
+
+#ifndef STATS_STRUCT_DEF
+#define STATS_STRUCT_DEF
+
+#include "ST_defines.h"
+
+/* Basic struct that holds average, sum of differences squared, standard deviation
+ * and number of entries. This is enough information to calculate running values without
+ * storing raw data. If you would like to create a new accumulator do not use this
+ * struct. Instead, use StatType.*/
+struct accumulators_st {
+  double ave, sum_dif_sqr, sd;
+  unsigned long nobs;
+};
+
+/* Accumulator along with the RGroup or Species name. */
+struct stat_st {
+  char *name; /* array of ptrs to names in RGroup & Species */
+  struct accumulators_st *s;
+} typedef StatType;
+
+/* Struct for wildfire and prescribed fire stats. */
+struct fire_st {
+  int *wildfire;
+  int **prescribedFire;
+} typedef FireStatsType;
+
+/*----------------------- Exported Functions ---------------------------------- */
+void stat_Collect( Int year ) ;
+void stat_Collect_GMort ( void ) ;
+void stat_Collect_SMort ( void ) ;
+void stat_Output_YrMorts( void ) ;
```

Comment:
Function is never used. Address in ST_stats.c as well unless we want to keep it for some reason.